### PR TITLE
added hysteresis flags

### PR DESCRIPTION
**Pull request by bogdan-rosianu**

### CODE REVIEW

File: cmd/filegen/main.go [hunk 1]
```diff
@@ -85,6 +85,15 @@ VERSION:
...
 		Value: 1,
 	}
+	hysteresis = cli.Float64Flag{
+		Name:  "hysteresis",
+		Usage: "Hysteresis value",
+		Value: 0.0,
+	}
+	adaptivity = cli.BoolFlag{
+		Name:  "adaptivity",
+		Usage: "adaptivity value",
```
Comment (AdoAdoAdo) [inline cmd/filegen/main.go]: true if shard merging and splitting is enabled

File: cmd/filegen/main.go [hunk 2]
```diff
@@ -85,6 +85,15 @@ VERSION:
 		Usage: "Number of initial metachain observers, private/public keys, to generate",
 		Value: 1,
 	}
+	hysteresis = cli.Float64Flag{
+		Name:  "hysteresis",
+		Usage: "Hysteresis value",
```
Comment (AdoAdoAdo) [inline cmd/filegen/main.go]: multiplied with numOfNodesPerShard to compute number of nodes allowed in the waiting list of each shard

File: cmd/filegen/main.go (hunk 1 above)
Comment (bogdan-rosianu) [reply cmd/filegen/main.go]: done

File: cmd/filegen/main.go (hunk 2 above)
Comment (bogdan-rosianu) [reply cmd/filegen/main.go]: done

**Pull request merged**